platform/broadcom/sonic-platform-modules-dell/z9332f/sonic_platform/sfp.py

```diff
@@ -1129,7 +1138,7 @@ def get_rx_power(self):
                 if rx_power_data is not None:
                      rx1_pw = self._strip_unit_from_str(rx_power_data['data']['RXPower']['value'])
                 else:
-                     rx1_pw = 0.0
+                     rx1_pw = "N/A"
```

Comment:
rx_power_data[] should be empty due to eeprom read failure. By appending 'N/A' the test will pass even though the eeprom read failed. The rx_power_list[] should be 'N/A' iff self.qsfp_dd_rxpower_supported is false.

Reply:
Done